xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuTranslations.fr.xml

```diff
@@ -77,7 +77,7 @@ menu.uix.scope.hint=Choix du contexte d'affichage du menu.
 
 # Menu Macro
 rendering.macro.menu.name=Menu
-### Missing: rendering.macro.menu.description=Displays a menu created using simple wiki syntax (nested lists and links).
+rendering.macro.menu.description=Affiche un menu créé en utilisant la syntaxe wiki simple (listes imbriquées et liens)
```

Comment:
Better: "en utilisant une syntax wiki simple". There's no such thing as "la syntaxe wiki simple" ;)

Reply:
JSDennebouy does not seems to have a github account and won't see this comment so better directly fix it on https://l10n.xwiki.org.

Reply:
@vmassol Feel free to make the change here: https://l10n.xwiki.org/translate/xwiki-platform/menu-application/fr/?checksum=d605bf29454b7e9b

Reply:
> JSDennebouy does not seems to have a github account and won't see this comment so better directly fix it on https://l10n.xwiki.org.

Yes I do !
Sorry Vincent, there may be some errors on technical translations.
The main modifications are changes of "mail" and "email" to "courriel".

Reply:
OK I see you associated the commits with your account, cool :)